src/internal/converters/eml/eml_test.go

```diff
@@ -461,7 +468,7 @@ func (suite *EMLUnitSuite) TestConvert_message_in_messageble_to_eml() {
 	assert.Equal(t, formatAddress(msg.GetFrom().GetEmailAddress()), eml.GetHeader("From"))
 
 	attachments := eml.Attachments
-	assert.Equal(t, 1, len(attachments), "attachment count in parent email")
+	assert.Equal(t, 3, len(attachments), "attachment count in parent email")
```

Comment:
Having a hard time finding which test case in testdata covers the bug fix here? Can you add a comment in the test data file?

Reply:
The test cases are at line 142(`TestConvert_edge_cases`). Basically the same tests we had for file attachments, we are running them now for item attachments as well. The testdata has 3 identical item attachments, with diff IDs. We `transform` the json testdata in `TestConvert_edge_cases` to overwrite attachment names for these test cases. 

I'll add a comment.

Reply:
Thanks. Does this include a case where the attachment doesn't have a name?

Reply:
Yep. Here https://github.com/alcionai/corso/pull/5209/files#diff-1932db8354803639e32f2bbaf0fbfba246fb2b2e3a6490f02a8e980d8413e003R171